docs/Activity.md

```diff
@@ -0,0 +1,385 @@
+# `interface Activity`
+
+`Activity` is a Microsoft Bot Framework representation of actions made by
+conversation participants.
+
+The full specification of the `Activity` interface can be found in the
+Microsoft Bot Framework SDK documentation at https://docs.microsoft.com/en-us/azure/bot-service/index-bf-sdk
+and in the Microsoft Bot Framework SDK specs at https://github.com/microsoft/botframework-sdk/tree/main/specs.
```

Comment:
TL;DR - Languages are mess and they rules don't exist.

In some degree, yes. Documentation is both countable and uncountable (thanks, English) and it this case we speak about a specific thing rather than abstract case. That is why I kept it there (it also sounded better to my ear while reading it with thick Scottish accent).

Specs on the other hand sound more like plural (specifications) than singular specification, so went with the no article in front of plural. However, the article can technically still stay if the Microsoft Bot Framework SDK specs is a particular name or something really specific.

Reply:
Changed now: https://github.com/yet-another-radek/botbuilder-adapter-tyntec-whatsapp/compare/e36cc18..2025655